docs/xxflightrecorder.md

```diff
@@ -0,0 +1,62 @@
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# -XX:[+|-]FlightRecorder
+
+**Linux&reg; on x86, Linux on AArch64**
+
+![Start of content that applies to Java 11 (LTS) and later](cr/java11plus.png) This option enables or disables the Java Flight Recorder (JFR) tool in the VM. This built-in profiling and troubleshooting tool in the VM collects profiling and diagnostic information.
+
+:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** Support for the JFR tool is currently provided as a technical preview. All JFR-related `jcmd` options might change in future releases.
+
+## Syntax
+
+        -XX:[+|-]FlightRecorder
+
+| Setting                            | Effect  | Default                                                                            |
+|------------------------------------|---------|:----------------------------------------------------------------------------------:|
+| `-XX:+FlightRecorder` | Enable  | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+| `-XX:-FlightRecorder` | Disable |               |
+
+## Explanation
+
+If the JFR tool is enabled in the VM with the `-XX:+FlightRecorder` option, then you can trigger profile and diagnostic recording with the [`jcmd`](https://eclipse.dev/openj9/docs/tool_jcmd/) tool. Recording does not start automatically, it must be triggered.
+
+To start a recording, specify the following command:
+
+```
+jcmd [JDK ID] JFR.start [filename=...] [duration=..s]
+
+```
+where:
+
+`filename=...` specifies the name of the file where the recording must be saved. This parameter is optional. If a file name is not specified, the recording is saved in the `defaultJ9recording.jfr` file in the process working directory by default.
+
+`duration=..` specifies the duration of the recording. Units are seconds (s), minutes (m), hours (h), and days (d). This parameter is optional. If the duration is not specified, the tool continues to record until the recording is manually stopped or the VM is shut down. ![End of content that applies to Java 11 (LTS) and later](cr/java_close.png)
```

Comment:
I think we should have an example. Can you specify, for example, both hours and minutes, how is this done? 1h30m?

Reply:
@tajila - Could you please provide an example of JFR `jcmd` command with both these parameters? Thanks!

Reply:
It only accepts 1 unit of time. Ie `12m`, `1h`, `54s` or `2d`